File: shared_code/templates/file_template.py
#!/usr/bin/env python3
"""
<file_description>

Created by: <agent_name>
Task ID: <task_id>
Date: <date>
"""

import os
import sys
from pathlib import Path

# Add parent directory to path for imports
sys.path.insert(0, str(Path(__file__).parent.parent.parent))

# Import shared modules if needed
# from shared_code.modules import module_name

class <class_name>:
    """<class_description>"""
    
    def __init__(self):
        """Initialize the <class_name>."""
        pass
    
    def method(self):
        """<method_description>"""
        pass

# Main function for testing
def main():
    """Main function for testing."""
    instance = <class_name>()
    print(f"Created instance of {instance.__class__.__name__}")

if __name__ == "__main__":
    main()

File: shared_code/templates/file_template.js
/**
 * <file_description>
 * 
 * Created by: <agent_name>
 * Task ID: <task_id>
 * Date: <date>
 */

// Import dependencies as needed
// const dependency = require('dependency');

/**
 * <class_description>
 */
class <class_name> {
  /**
   * Create a new instance
   */
  constructor() {
    // Initialize properties
  }

  /**
   * <method_description>
   * @param {type} param - description
   * @returns {type} description
   */
  method(param) {
    // Implementation
    return param;
  }
}

// Export the class for use in other modules
module.exports = <class_name>;

// For testing
if (require.main === module) {
  const instance = new <class_name>();
  console.log(`Created instance of ${instance.constructor.name}`);
}